packages/bot-web-ui/src/constants/contract.js

```diff
@@ -0,0 +1,225 @@
+import { localize } from '@deriv/translations';
+
+export const card_labels = Object.freeze({
```

Comment:
This should be a function (`getCardLabels`, like `getMarketNamesMap`) as `localize` will only be executed once for each value here. If the user switches the language we should update the value.